addons/planner_project/data/planner_data.xml

```diff
@@ -0,0 +1,1211 @@
+                                <li><span>#2</span><textarea id="input_element_act2" value="" line="1" type="text"/></li>
+                                <li><span>#3</span><textarea id="input_element_act3" value="" line="1" type="text"/></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+
+               <div id="planner_page4" class="planner-page">
+                    <h1>Your Projects </h1>
+                    <p class="text-muted">Depending on what you need and how you want to manage, there are several ways to do things in Odoo. First, choose if you want to think in terms of tasks or issues. Then, activate Timesheets if you need them.</p>
+                    <h4>1. Learn about Tasks, Issues and Timesheets.</h4>
+                    <section class="container-fluid">
+                        <div class="row">
+                            <div class="col-md-12">
+                                <div class="panel panel-success">
+                                    <div class="panel-heading">
+                                        <span class="panel-title">
+                                            <span class="fa fa-check-square-o"/>
+                                            <strong> Tasks</strong>
+                                        </span>
+                                     </div>
+                                     <div class="panel-body">
+                                        <p><strong>An internal activity</strong> that should be done within a defined period of time.</p>
+                                        <p><strong>Example: </strong></p>
+                                        <p>During a meeting, a customer asks a manager for a few modifications to a project.
+                                        </p>
+                                        <p>Back at the office, the manager splits the customer's demands into several tasks and delegates them to several employees.</p>
+                                        <p>Each employee will have their own simple task, while the manager will be able to follow the global advancement through the principal task.</p>
+                                        <p>Tasks are the main mechanism in Odoo and are activated by default.</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-md-12">
+                                <div class="panel panel-success">
+                                    <div class="panel-heading">
+                                        <span class="panel-title">
+                                            <span class="fa fa-exclamation-circle"/>
+                                            <strong> Issues</strong>
```

Comment:
As 'panel-title' class doesn't have bold attribute. So i have use strong tag. Should i remove strong tag?